ledger/src/blockstore.rs

```diff
@@ -1257,6 +1259,21 @@ impl Blockstore {
                 metrics.num_coding_shreds_invalid += 1;
                 return false;
             }
+
+            if let Some(merkle_root_meta) = merkle_root_metas.get(&erasure_set) {
+                // A previous shred has been inserted in this batch or in blockstore
+                // Compare our current shred against the previous shred for potential
+                // conflicts
+                if !self.perform_merkle_check(
```

Comment:
`perform_merkle_check` is a pretty confusing name and error prone.
Like by no means it is clear if `false` is good or bad.